src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj

```diff
@@ -52,8 +52,21 @@
     <MauiSplashScreen Include="Resources\Images\dotnet_bot.svg" Color="#FFFFFF" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Pages\HitTestingPage.xaml.cs">
+      <SubType>Code</SubType>
+      <DependentUpon>HitTestingPage.xaml</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
   <Import Condition=" '$(UseMaui)' != 'true' " Project="..\..\..\BlazorWebView\src\Maui\build\Microsoft.AspNetCore.Components.WebView.Maui.targets" />
 
   <Import Project="../../../Maui.InTree.props" Condition=" '$(UseMaui)' != 'true' " />
 
+  <ItemGroup>
```

Comment:
Are not necessary changes although we can clean the csproj later as well.

Reply:
Fixed in a1a7afd